boskos/resources.json

```diff
@@ -187,7 +183,11 @@
       "k8s-jkns-gce-ubuntu-1-6",
       "k8s-jkns-e2e-gce-ubuntu-serial",
       "k8s-jkns-e2e-gce-ubuntu-slow",
-      "k8s-jkns-e2e-gce-ubuntu"
+      "k8s-jkns-e2e-gce-ubuntu",
+      "ci-kubernetes-e2e-gke-gpu",
```

Comment:
I would bet someone created the project without enable it for gke and immediately abandoned it...